canonical/canonical.com
# Send withdrawal confirmation email

## Done

- Send withdrawal confirmation email with a link 

## QA

- Go to https://canonical-com-593.demos.haus/careers/application/token (Please ping me for token) 
- Click application withdrawal button.
- Type `min.kim@canoinical.com` to send confirmation email. (Also, try any other email to check it's not sending email when the email doesn't match)
- Check the printed email in your terminal.
- Copy the link triggering application withdrawal.
- Check the printed result in the terminal. 
- [Spec](https://docs.google.com/document/d/1aENPmLdPjoRtMgmmGX2x4iewevRv905OdkKgqqm-sHk/edit)
- [Miro](https://miro.com/app/board/uXjVO9E_QFA=/?share_link_id=997105033759) for UX
- [Figma](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?node-id=4563%3A8766) for design

## Issue / Card

Fixes [#583](https://github.com/canonical/canonical.com/issues/583), [#581](https://github.com/canonical/canonical.com/issues/581), #596, [#585](https://github.com/canonical/canonical.com/issues/585)
